src/lib/eina/eina_debug.c

```diff
@@ -556,14 +556,17 @@ _monitor(void *_data)
 
    // set a name for this thread for system debugging
 #ifdef EINA_HAVE_PTHREAD_SETNAME
+/*
 # ifndef __linux__
    pthread_set_name_np
 # else
    pthread_setname_np
 # endif
      (pthread_self(), "Edbg-mon");
-#endif
 
+*/
+#endif
```

Comment:
Why not removing this?

Reply:
just not to forget how it was, i will fix these comments in a new commit

Reply:
Maybe you should add it to [issue 3](https://github.com/expertisesolutions/efl/issues/3), so we deal with it latter.